src/resources/users/user.model.ts

```diff
@@ -0,0 +1,32 @@
+import { v4 as uuid } from 'uuid';
+import { IUser } from './user.interface';
+
+export default class User {
+  private readonly id: string;
+
+
+
```

Comment:
We don't need so many empty lines in a row :)

Reply:
sorry. removed